From d381ac8addfc8ac46cf76faffcc06765de13097c Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 10 Jan 2025 17:10:59 +0100
Subject: [PATCH 1445/1695] dts: stm32: remove always-on from vdd33ucpd in
 stm32mp257f-dk.dtsi

Remove always-on property from "vdd33ucpd" node and its parent "vdd3v3_usb"
On stm32mp257f-dk board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 09f0ee65233254411402fa6d4df45e0a2d44c302 dts: stm32: remove always-on from vdd33ucpd in stm32mp257f-dk.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ia0a6ced170509da835310324ff23e69cbb6a712d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/428239
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 7ccdda94b..f464b5d28 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -273,7 +273,6 @@
 				regulator-name = "vdd3v3_usb";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
 				regulator-over-current-protection;
 			};
 			v5v_hdmi: ldo5 {
@@ -337,7 +336,6 @@
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vdd33ucpd-supply = <&vdd3v3_usb>;
-		regulator-always-on;
 	};
 
 	vdda18adc: vdda18adc {
-- 
2.47.2

